sphinx/directives/code.py

```diff
@@ -170,7 +169,7 @@ def run(self):
             try:
                 literal = container_wrapper(self, literal, caption)
             except ValueError as exc:
-                return [document.reporter.warning(text_type(exc), line=self.lineno)]
+                return [document.reporter.warning(str(exc), line=self.lineno)]
```

Comment:
I found `Reporter` takes an exception object as its argument:
https://github.com/docutils-mirror/docutils/blob/e88c5fb08d5cdfa8b4ac1020dd6f7177778d5990/docutils/utils/__init__.py#L152-L160

So this conversion is not needed.

Reply:
Done in #5813